pod.go

```diff
@@ -17,14 +17,17 @@
 package virtcontainers
```

Comment:
nit: in your commit message you say:
```
If we find a valid path for a kernel
or an image, we change the hypervisor config to point to them.
```

The `if` bothers me - if half implies that if the paths are not valid then we don't change the hypervisor (and carry on??) - presumably if there is an invalid config/annotation we will actually error out and not continue to launch the workload with the defaults?